src/Appwrite/Utopia/Database/Validator/Queries/Users.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+use Appwrite\Utopia\Database\Validator\Queries\Collection;
+
+class Users extends Collection
+{
+    public const ALLOWED_ATTRIBUTES = [
+        'name',
+        'email',
+        'phone',
+        'status',
+        'passwordUpdate',
+        'registration',
+        'emailVerification',
+        'phoneVerification',
```

Comment:
Let's please double-check we have indexes in collections.php for all these attributes

Reply:
All good, I double-checked.